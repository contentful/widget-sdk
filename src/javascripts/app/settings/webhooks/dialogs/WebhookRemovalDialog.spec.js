import React from 'react';
import { shallow } from 'enzyme';
import WebhookRemovalDialog from './WebhookRemovalDialog.es6';

describe('webhooks/dialogs/WebhookRemovalDialog', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <WebhookRemovalDialog
        isShown
        isConfirmLoading={false}
        onCancel={() => {}}
        onConfirm={() => {}}
        webhookUrl="https://test.com"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
