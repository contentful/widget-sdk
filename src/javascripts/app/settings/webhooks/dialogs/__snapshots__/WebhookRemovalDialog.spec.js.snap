// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`webhooks/dialogs/WebhookRemovalDialog should match snapshot 1`] = `
<ModalConfirm
  cancelLabel="Don't remove"
  cancelTestId="remove-webhook-cancel"
  confirmLabel="Remove"
  confirmTestId="remove-webhook-confirm"
  intent="negative"
  isConfirmDisabled={false}
  isConfirmLoading={false}
  isShown={true}
  onCancel={[Function]}
  onConfirm={[Function]}
  shouldCloseOnEscapePress={true}
  shouldCloseOnOverlayClick={true}
  size="medium"
  testId="cf-ui-modal-confirm"
  title="Remove Webhook"
>
  <p>
    You are about to remove webhook calling the following URL:
  </p>
  <div
    style={
      Object {
        "fontFamily": "monospace",
        "fontSize": ".9em",
        "margin": "10px 0",
        "wordWrap": "break-word",
      }
    }
  >
    https://test.com
  </div>
  <p>
    After removal your external integrations may stop working properly. Do you want to proceed?
  </p>
</ModalConfirm>
`;
